src/iocore/net/UnixUDPNet.cc

```diff
@@ -97,7 +97,7 @@ UDPPacket::new_UDPPacket(struct sockaddr const *to, ink_hrtime when, Ptr<IOBuffe
   p->p.segment_size = segment_size;
 #ifdef HAVE_SO_TXTIME
   if (send_at_hint) {
-    memcpy(&p->p.send_at, &send_at_hint, sizeof(struct timespec));
+    memcpy(&p->p.send_at, send_at_hint, sizeof(struct timespec));
```

Comment:
Ohh!! this a very good catch.
Must have done some refactor before pushing this and let this slide!! 

Please split this into a different PR just in case.

Reply:
Ok. Will close this PR and make new ones.